docs/api_refs/package.json

```diff
@@ -28,6 +28,7 @@
     "postcss": "^8",
     "tailwindcss": "^3.3.0",
     "ts-morph": "^20.0.0",
+    "typedoc": "^0.25.6",
```

Comment:
Why was this not here before?

Reply:
We were using npx b/c I didn't want to add a dep (don't remember my reasoning), but I think this is a better practice.